public/mock-backend/public/screenshot.png.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
```

Comment:
Didn't we determine that this is unnecessary?

Reply:
ARGHH! rebase issues everywhere -_-

Reply:
fixed